tests/kernel/gen_isr_table/src/main.c

```diff
@@ -50,6 +50,13 @@ extern uint32_t _irq_vector_table[];
  * the board.
  */
 #define TEST_NUM_IRQS	30
+#elif defined(CONFIG_SOC_NPCX7M6FB)
+/* In NPCX7M6FB, it uses some of the IRQs at the end of the vector table, for
+ * example, the irq 60 and 61 used for Multi-Input Wake-Up Unit (MIWU) device
+ * by default, and conflicts with isr used for testing. Moving IRQs for this
```

Comment:
Done. The other approach is setting all miwu devices statuses in device tree as 'disabled'. It won't install ISRs such as IRQ 60 and 61 by default in this test suite. But you need to enable them in a board device tree file. For example, you need to add the following declarations in npcx7m6fb.dts file.

_&miwu0 {
	status = "okay";
};
&miwu1 {
	status = "okay";
};
&miwu2 {
	status = "okay";
};_

It's crucial to enable all MIWU devices by default since lots of devices' interrupt handler relies on them. Adding them in all boards' device tree files use npcx ec is necessary.